src/init.js

```diff
@@ -41,6 +42,8 @@ function init(options) {
     var fileContent = file.contents.toString();
     var sourceMap;
 
+    file.newline = detectNewline.graceful(fileContent || '');
```

Comment:
Good to know ok, so make this sourceMapExt field or gulpSourceMap because I will use the same object in another PR I am working on.